### PR TITLE
Added text, links and pictures to about us page and added placeholder htmls

### PR DESCRIPTION
**Pull request by mojcas**
This is to address issues #32, #33 and #36 
Added text link to the footer in base.html, changed text in about.html and added links to Twitter, FB and the "OTHER" page :)
Added placeholder htmls to address issues #13 #16 #20 #22 and fixed links

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -51,13 +51,12 @@
...
 		<footer class="footer">
-				<div class="container">
+				<div class="container">	
 					<a href="https://github.com/CodeCatz">
-					<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<p>GITHUB CodeCatz: Kittens learning to code like Catz</p>
 					</a>
-				</div>
+				</div>
```
Comment (goranche) [inline templates/base.html]: would you mind removing the trailing whitespace? :wink: 
there are tabs after the </div>

also, this whole div (and it's content) might be indented one level less (to the left) but I will not hold that against you :angel:

Comment (mojcas) [inline templates/base.html]: Done.. I've done some more commits, should I create a new pull request?
:)

Comment (ialja) [inline templates/base.html]: Good job! New commits will automatically be added to the Pull request if you have one open :)

Comment (goranche) [inline templates/base.html]: no, it's not done, at least not in this pull request :wink: 
any additional commits you push into your repository, will be added to this pull request

so you might not want to work on other stuff before this get's merged :smile: 
we'll have to show you how to create feature branches, I guess... :thought_balloon:

Comment (goranche) [inline templates/base.html]: oh, never mind... you've pushed... :grin:
ok, now for some more reviewing

**Comment by mojcas** [comment]
Added placeholder htmls to address issues #13 #16 #20 #22

File: templates/pages/events.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Events{% endblock %}
+
+{% block content%}
+
+
+	<div>
+		<h1><a href="">All CodeCatz Events</a></h1>
```
Comment (goranche) [inline templates/pages/events.html]: why are all headers links, that point to nowhere?

Comment (mojcas) [inline templates/pages/events.html]: you know @goranche I was asking myself exact same question.. :smiley:

Comment (goranche) [inline templates/pages/events.html]: :smile: here, have a :lollipop: :wink: 

just remove the a tags (and closing tags) and I'll merge it :+1: 
good work

Comment (mojcas) [inline templates/pages/events.html]: Done @goranche  :+1:

**Comment by goranche** [comment]
love your commit messages, keep it up :+1:

**Comment by mojcas** [comment]
Thank you ! :turtle: :laughing:

**Comment by goranche** [comment]
you are very welcome... and thank *you* :wink:

**Comment by ialja** [comment]
You rock, @mojcas! :tada:

**Pull request merged**